js/nodes/materials/structural_material.js

```diff
@@ -0,0 +1,64 @@
+// TODO: Create a material base class
+class StructuralMaterial {
+    constructor()
+    {
+        this.addInput("model_part_name","string");
+        this.properties = {
+            "model_part_name" : "Structure.Parts_Solid_Solid_Auto1",
+            "properties_id"   : 1,
+            "Material"        : {
+                "constitutive_law" : {
+                    "name" : "LinearElasticPlaneStress2DLaw"
```

Comment:
This must be an input.

Reply:
@rubenzorrilla these are the defaults, assignments are done in `onExecute` function.

Reply:
OK, I missed the part in which the claw is chosen. I thought it was hardcoded in the defaults.